File: app/src/main/java/b12app/vyom/com/bookmybus/view/Routes/RoutesContract.java
package b12app.vyom.com.bookmybus.view.Routes;

import android.content.Context;

import java.util.List;

import b12app.vyom.com.bookmybus.model.JBusByRoute;

public interface RoutesContract {

    interface IView{
      void  initRecyclerView(List<JBusByRoute.BusinformationBean> businformationBeanList);
      
    }

    interface IPresenter{



        void getBusesForRoute();
    }
}

File: app/src/main/java/b12app/vyom/com/bookmybus/view/routes/RoutesContract.kt
package b12app.vyom.com.bookmybus.view.routes

import b12app.vyom.com.bookmybus.model.JBusByRoute

interface RoutesContract {

    interface IView {
        fun initRecyclerView(businformationBeanList: List<JBusByRoute.BusinformationBean>,startCity:String,endCity:String)

    }

    interface IPresenter {


        fun getRouteId(startLatitude: String, startLongitude: String, endLatitude: String, endLongitude: String)
        fun getBusesForRoute(id: String,startCity:String,endCity:String)
    }
}
